### ISSUE TITLE
A11y_SQLAzureDataStudio_Query_Create&Run_ScreenReader: Data present on the screen not read by screen reader.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9229]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#DesktopApp;#Win32;#MAS1.3.1; #Benchmark;#A11y_SQLAzureDataStudio;#A11ySev2;#JAWS;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Screen Reader: JAWS
MAS reference: MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio.
2. Navigate to File > New Query
3. Connect to a Server in the Query Editor and choose the credential, and optionally the database to connect to the server.
4. Type a SQL query statement.
5. Click Run.
6. Navigate to Messages section.
7. Check if content is properly read by screen reader.

**Actual:**
Content present on the screen below messages section is not read by JAWS screen reader.

**Expected:**
All the content present on the screen should be read by the screen reader.

**User Impact:**
Screen reader users might receive less information of what is present on the screen.

**Attachment link reference:**
[9229_A11y_SQLAzureDataStudio_Query_Create&Run_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4291428/9229_A11y_SQLAzureDataStudio_Query_Create.Run_ScreenReader.zip)

**Comment by alanrenmsft** [comment]
dupe of #9408

**Comment by Harika1702** [comment]
https://user-images.githubusercontent.com/100270074/219005258-1eee9592-5448-4b83-8825-550a2e33ac8a.mp4

Verified this issue now issue is not repro'ing.
Version: 1.41.1 (user setup)
Commit: 33dce95b152227c4f10f6251ab30690fe9895c0c
Date: 2023-01-28T00:46:39.274Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22623
GitHub Tags: #Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
